book-example/src/continuous-integration.md

```diff
@@ -87,3 +87,92 @@ deploy: book
+
+```yaml
+name: Generate ebook
+
+on: 
+  push:
+
+jobs:
+  ebook:
+    name: Build and upload
+    runs-on: ubuntu-latest
+    steps:
+      # Install Rust & its package manager Cargo
+      - name: Install cargo
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+
+      # Cache installation assets
+      - name: Cache cargo registry
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+      - name: Cache cargo index
+        uses: actions/cache@v1
+        with:
+          path: ~/.cargo/git
+          key: ${{ runner.os }}-cargo-index-${{ hashFiles('**/Cargo.lock') }}
+      - name: Cache cargo build
+        uses: actions/cache@v1
+        with:
+          path: target
+          key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
+
+      # Install mdbook and requirements
+      - name: Install mdbook and epub plugin
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: mdbook mdbook-epub
+
+      # Ebook generation
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Here you can add one or more steps, like generating a summary or whatever
+      # - name: Generate summary
+      #   run: ./generate_summary.sh
+
+      - name: Generate ebook from markdown
+        run: mdbook build
+
+      # ePub upload as a Github release asset
+      - name: Create Release
+        id: create-release
+        uses: actions/create-release@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+
+      - name: Upload epub to release
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create-release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
+          asset_path: ./book/epub/Songbook.epub
+          asset_name: songbook.epub
+          asset_content_type: application/epub+zip
+
+      # HTML publication as Github Page
+      - name: Publish HTML
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./book/html
+          publich_branch: ${{ github.ref }}
```

Comment:
Oh, slight typo here?

Reply:
Good catch, I'll fix it as soon as my laptop end up booting